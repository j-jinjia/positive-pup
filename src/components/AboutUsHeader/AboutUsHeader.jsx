import Hero from "../Hero/Hero";
import "./AboutUsHeader.scss";

const AboutUsHeader = () => {
  return (
    <div className="about-us-header">
      <Hero
        header={
          <>
            <h2 className="hero__header">Hey there</h2>
            <h2 className="hero__header">My name is Melanie!</h2>
          </>
        }
        subheaderText={
          "As the owner of The Positive Pup, it is my joy to be able to welcome you to this website."
        }
        buttonText={"More About Us"}
        isSecondary={true}
      />
    </div>
  );
};

export default AboutUsHeader;
