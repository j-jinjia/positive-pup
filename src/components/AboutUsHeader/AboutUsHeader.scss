@use "../../styles/variables" as *;

.about-us-header {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  gap: 50px;
  align-items: center;
  padding: 2rem;

  &__image {
    width: 100%;
    max-width: 430px;
  }
  &__text {
    color: $colour-font-paragraph;
    padding-bottom: 2rem;
  }
}

@media screen and (min-width: $desktop-breakpoint) {
  .about-us-header {
    &__image {
      max-width: 540px;
      padding-right: 5rem;
    }
    &__content {
      max-width: 45%;
      padding-left: 5rem;
    }
  }
}
