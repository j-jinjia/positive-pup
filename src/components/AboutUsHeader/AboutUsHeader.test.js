import { customRender } from "../../utils/testUtils";
import AboutUsHeader from "./AboutUsHeader";

it("should match AboutUsHeader component snapshot", () => {
  const { container } = customRender(<AboutUsHeader />);

  expect(container).toMatchSnapshot();
});
