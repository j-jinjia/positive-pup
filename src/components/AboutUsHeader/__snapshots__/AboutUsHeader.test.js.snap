// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match AboutUsHeader component snapshot 1`] = `
<div>
  <div
    class="about-us-header"
  >
    <main
      class="about-us-header__content"
    >
      <h1
        class="about-us-header__content-title"
      >
        ABOUT US
      </h1>
      <div
        class="hero"
      >
        <h2
          class="hero__header"
        >
          Hey there
          <div>
             
          </div>
          My name is Melanie!
        </h2>
        <p
          class="hero__subheader"
        >
          As the owner of The Positive Pup, it is my joy to be able to welcome you to this website.
        </p>
        <button
          class="button button--secondary"
        >
          More About Us
        </button>
      </div>
    </main>
    <img
      alt="Melanie and a dog"
      class="about-us-header__image"
      src="AboutMe.png"
    />
  </div>
</div>
`;
